standard/statements.md

```diff
@@ -313,6 +313,11 @@ Implicitly typed declarations contain the contextual keyword ([§6.4.4](lexical-
 - If there is no type named `var` in scope and the input matches *implicitly_typed_local_variable_declaration* then it is chosen;
 - Otherwise if a type named `var` is in scope then *implicitly_typed_local_variable_declaration* is not considered as a possible match.
 
+For reference types and type parameters that aren't constrained to be a value type, `var` infers an annotated type. For example:
```

Comment:
Did we define "annotated type"? I thought the preferred term was "nullable type"